travis-ci/managers/debian.sh

```diff
@@ -3,15 +3,20 @@
 PHASES=(${@:-SETUP RUN RUN_ASAN CLEANUP})
 DEBIAN_RELEASE="${DEBIAN_RELEASE:-testing}"
 CONT_NAME="${CONT_NAME:-debian-$DEBIAN_RELEASE-$RANDOM}"
-DOCKER_EXEC="${DOCKER_EXEC:-docker exec -it $CONT_NAME}"
+ENV_VARS="${ENV_VARS:-}"
 DOCKER_RUN="${DOCKER_RUN:-docker run}"
 REPO_ROOT="${REPO_ROOT:-$PWD}"
-ADDITIONAL_DEPS=(clang)
+ADDITIONAL_DEPS=(clang pkg-config)
+CFLAGS="-g -O2 -Werror -Wall"
 
 function info() {
     echo -e "\033[33;1m$1\033[0m"
 }
 
+
+function docker_exec() {
```

Comment:
Do you know why `DOCKER_EXEC` was set via variable that could be overridden via env var? (`${DOCKER_EXEC:-...}`). Is it required by Travis CI to be able to override it or not really?

Reply:
@rdna `DOCKER_EXEC` is initially set in the upper level config: https://github.com/libbpf/libbpf/blob/master/.travis.yml#L74
From what I see it's not required, looks like it can also be removed from `.travis.yml`.

Reply:
Just a bit of context. Historically those scripts were over-engineered. Although they were simplified a bit (by removing unnecessary steps like creating images and sending them to Docker Hub) they are still kind of opaque and I don't always understand what they are trying to do and why as in https://github.com/systemd/systemd/pull/12402#issuecomment-486765499. In my opinion most of those environment variables (including DOCKER_EXEC) can be safely removed without losing anything (and probably making the scripts easier to understand and run manually).

Reply:
`DOCKER_EXEC` is removed.